cmd/algorelay/relayCmd.go

```diff
@@ -108,52 +103,62 @@ func loadRelays(file string) []eb.Relay {
 }
 
 type checkResult struct {
-	ID        int64
-	Success   bool
-	Error     string  `json:",omitempty"`
+	ID      int64
+	Success bool
+	Error   string `json:",omitempty"`
 }
 
 type dnsContext struct {
-	nameEntries    map[string]string
-	bootstrap      srvService
-	metrics        srvService
+	nameEntries map[string]string
+	bootstrap   srvService
+	metrics     srvService
+	srvZoneID   string
+	nameZoneID  string
 }
 
 type srvService struct {
-	serviceName  string
-	entries      map[string]uint16
-	shortName    string
-	networkName  string
+	serviceName string
+	entries     map[string]uint16
+	shortName   string
+	networkName string
 }
 
 func makeDNSContext() *dnsContext {
-	nameEntries, err := getReverseMappedEntries(cfNameZoneID, nameRecordTypes)
+	nameEntries, nameZoneID, err := getReverseMappedEntries(nameDomainArg, nameRecordTypes)
```

Comment:
For consistency, can you revert the call to getReverse...() to take zoneID and do the NewCred/ZoneID lookup here (like you do for srvZoneID)?

Reply:
yes